src/isomorphism/vf2.rs

```diff
@@ -10,74 +10,90 @@
 // License for the specific language governing permissions and limitations
 // under the License.
 
+#![allow(clippy::too_many_arguments)]
 // This module is a forked version of petgraph's isomorphism module @ 0.5.0.
 // It has then been modified to function with PyDiGraph inputs instead of Graph.
```

Comment:
I would probably reword this comment now to something like

```suggestion
// This module was originally forked from petgraph's isomorphism module @ v0.5.0
// to handle PyDiGraph inputs instead of petgraph's generic Graph. However it has
// since diverged significantly from the original petgraph implementation.
```

since the petgraph lineage is still there but it's not longer just about handling `PyDiGraph`,
well specifically `StableGraph<PyObject, PyObject, Ty>` (the old petgraph isomorphism
only worked with `Graph`) and handling `PyResult` on the return for the semantic matcher
as we've added a lot of new functionality here since I originally forked it.